src/pages/home/report/ReportActionCompose.js

```diff
@@ -565,11 +566,13 @@ class ReportActionCompose extends React.Component {
                             </>
                         )}
                     </AttachmentModal>
-                    <EmojiPickerButton
-                        isDisabled={isBlockedFromConcierge || isArchivedChatRoom}
-                        onModalHide={() => this.focus(true)}
-                        onEmojiSelected={this.addEmojiToTextBox}
-                    />
+                    {canUseTouchScreen() && !this.props.isSmallScreenWidth ? null : (
```

Comment:
@thesahindia Thank you for bringing this up. Maybe we could come back to try using this
`{canUseTouchScreen() && this.props.isMediumScreenWidth ? null : (`

Reply:
We can do that but it won't hide the emoji picker in portrait mode for tablets that have smaller screens

Reply:
That is acceptable I think, in portrait it wont be bad! Will you submit a PR and link the OG issue so it picks me for review? Thank you very much @thesahindia 🙇

Reply:
Yeah that's acceptable.

Reply:
Yeah sure, will do it soon.

Reply:
Ahh got a problem if we use mediumScreenWidth then it will not remove the emoji picker menu for tablets with bigger screens (tested on 11 inch iPad)

Reply:
So.. new breakpoint for ipad pro 12.9 inch ?

Reply:
Hmm I don't think that would be a great idea because there's another problem, the emoji picker won't show up if user decreases his screen size in touch screen laptop.

Reply:
At this point, I think it'll be easier to just dismiss keyboard when emoji picker is opened.

This won't work for mWeb on tablets. So we gotta choose our tradeoffs

Reply:
Let's just add the mediumScreenWidth. The problem is not 11 inch ipad but smaller tablets like ipad mini or even smaller android tablets